test/api.jl

```diff
@@ -8,7 +8,7 @@ import Pkg.Types.PkgError, Pkg.Resolve.ResolverError
 using UUIDs
 
 using ..Utils
-
+original_path = pwd()
 @testset "Pkg.activate" begin
     temp_pkg_dir() do project_path
         cd_tempdir() do tmp
```

Comment:
The `cd_tempdir()` shoudl already do this I think.

Reply:
What changes the active directory in these tests?

Reply:
xref https://github.com/JuliaLang/Pkg.jl/pull/2089 which was never fixed

Reply:
I wouldn't mind if someone more knowledgeable took this PR over